File: Application/api/src/main/java/com/wordflip/api/models/pupil.java
package com.wordflip.api.models;

import java.util.Calendar;

/**
 * Created by fhict on 27-05-16.
 */
public class Pupil {
    private int id;
    private String username;
    private String password;
    private Calendar lastLoggedIn;

    public Pupil(int id, String username, String password, Calendar lastLoggedIn) {
        this.id = id;
        this.username = username;
        this.password = password;
        this.lastLoggedIn = lastLoggedIn;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public Calendar getLastLoggedIn() {
        return lastLoggedIn;
    }

    public void setLastLoggedIn(Calendar lastLoggedIn) {
        this.lastLoggedIn = lastLoggedIn;
    }
}

File: Application/WordFlip/WordFlip/Pupil.swift
//
//  Pupil.swift
//  LoginTrial
//
//  Created by Fhict on 02-06-16.
//  Copyright © 2016 Fontys. All rights reserved.
//

import Foundation

class Pupil
{
    var id: Int
    var userName: String
    var passWord: String
    var lastLoggedIn: NSDate
    
    init(id: Int, userName: String, passWord: String, lastLoggedIn: NSDate){
        self.id = id
        self.userName = userName
        self.passWord = passWord
        self.lastLoggedIn = lastLoggedIn
    }
}